### DISCUSSION TITLE
The correct value of MinRxSymbols?

### DISCUSSION THREAD

**Discussion by kelah** [Lora-net/LoRaMac-node#1596] [Q&A] [answered]
Hi,
 Hope you have a good day. I found out that  min RxSymbols is 6

'''
Nvm.MacGroup2.MacParamsDefaults.MinRxSymbols = 6;
'''

However, "SX1276 Settings for LoRaWAN" document specfied that it should be minimum 5. Any reason why? Thanks!

**Comment by mluis1** [comment, marked as answer]
To detect a LoRa signal only 5 symbols are required. At least in theory.
However, there are always some tolerance to take account for.

From experiments it was found that setting the minimum value to 6 improved the chances to not miss a downlink.

You may try to change the value to 5 and see if it suits your requirements. But I would recommend to keep the 6 symbols value.
The impact on the power consumption is minimal and ensures a more reliable detection and reception of a downlink.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Reply by kelah** [reply]
ok thanks!!
